codice/alliance
# CAL-41 Provide the ability to drape NITF overviews on the map

#### What does this PR do?
Adds the ability to drape NITF overviews on the Cesium and OpenLayers maps.

This PR different from https://github.com/codice/ddf/pull/699 in that it only exposes the NITF overview as an overlayable image. It doesn't create any new UI components, action providers, or metacard transformers, as those were developed in the DDF ticket.

#### Who is reviewing it?
@glenhein 
@dcruver 
@kcwire 
@jlcsmith 

#### How should this be tested?
Ingest some NITFs. Go to the Search UI and search for them. Verify that each NITF with an overview image has a Map Action called `Overlay overview on the map`, and that when it is clicked, the overview image is overlaid on the map.

#### What are the relevant tickets?
[CAL-41](https://codice.atlassian.net/browse/CAL-41)

#### Screenshots (if appropriate)
See CAL-41

#### Checklist:
